code/__HELPERS/time.dm

```diff
@@ -23,15 +23,27 @@
 	return wtime + (time_offset + wusage) * world.tick_lag
 
 //Returns the world time in english
-/proc/worldtime2text(time = world.time)
-	time = (round_start_time ? (time - round_start_time) : (time - world.time))
-	return "[round(time / 36000)+12]:[(time / 600 % 60) < 10 ? add_zero(time / 600 % 60, 1) : time / 600 % 60]"
-
-/proc/time_stamp()
-	return time2text(world.timeofday, "hh:mm:ss")
-
-/proc/gameTimestamp(format = "hh:mm:ss") // Get the game time in text
-	return time2text(world.time - timezoneOffset + 432000, format)
+/proc/worldtime2text()
+	return gameTimestamp("hh:mm:ss", world.time)
```

Comment:
So, I'm kinda considering reverting this change - worldtime2text no longer necessarily starts at 00:00. This is because gameTimestamp subtracts `timezoneOffset`, which I would guess is machine dependent... I'm not sure why you would want to offset the _relative time dream daemon has been running_ by the time zone of the server, though.

Reply:
Update: The reason this happens is, uh, well, complicated to say the least.

However, it's simple to fix: Disable daylights savings time on your system clock